module/localisation/RobotParticleLocalisation/src/RobotParticleLocalisation.cpp

```diff
@@ -239,13 +239,13 @@ namespace module::localisation {
 
         on<Startup, With<FieldDescription>>().then([this](const FieldDescription& fd) {
             // Left side penalty mark
-            config.start_state.emplace_back((-fd.dimensions.field_length / 2.0) + fd.dimensions.penalty_mark_distance,
-                                            (-fd.dimensions.field_width / 2.0),
+            config.start_state.emplace_back((-fd.dimensions.field_length / 2.0f) + fd.dimensions.penalty_mark_distance,
+                                            (-fd.dimensions.field_width / 2.0f),
```

Comment:
Multiplications are cheaper than divisions, try to use them if it is convenient
```suggestion
            config.start_state.emplace_back((-fd.dimensions.field_length * 0.5f) + fd.dimensions.penalty_mark_distance,
                                            (-fd.dimensions.field_width * 0.5f),
```